EventsExpress/ClientApp/src/components/event/event-form.js

```diff
@@ -14,25 +14,58 @@ import {
     renderSelectPeriodicityField,
     renderCheckbox,
     renderTextField,
-    renderDatePicker
+    renderDatePicker,
+    radioLocationType
 } from '../helpers/helpers';
 import Inventory from '../inventory/inventory';
 import LocationMap from './map/location-map';
+import { enumLocationType } from '../../constants/EventLocationType';
 
 momentLocaliser(moment);
 const { validate } = Module;
 
 const required = value => value ? undefined : "Required";
 
-class EventForm extends Component {
+class EventForm extends Component {    
 
-    state = { checked: false };
+    state = { checked: false,  mapChecked: true, onlineChecked: false };
+
+    componentDidMount = () => {
+        if (this.props.initialValues != undefined) {
+            if (this.props.initialValues.location.onlineMeeting) {
+                this.setState(state => ({
+                    mapChecked: false,
+                    onlineChecked: true,
+                }));
+            }
+            else {
+                this.setState(state => ({
+                    mapChecked: true,
+                    onlineChecked: false,
+                }));
+            }
+        }
+    }
 
     handleChange = () => {
         this.setState(state => ({
             checked: !state.checked,
         }));
     }
+    handleMapLocationChange = () => {
```

Comment:
These two callbacks are not used anywhere. Consider removing them togather with state.